src/components/shared/GradientBox.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function GradientBox({
+  children,
+  from,
+  to,
+  gradientOn = false,
+  className = "",
+}) {
+  return (
+    <div
+      className={`bg-gradient-to-br ${
+        gradientOn ? `from-[${from}] to-[${to}]` : "md:bg-none"
+      } items-center justify-center flex flex-col gap-5 p-10 ${className}`}
+    >
```

Comment:
The gradient box doesnt work, it seems tailwind isn't registering the props.

![Screenshot 2024-07-02 at 09 53 54](https://github.com/thedxberience/dxberience-frontend-v2/assets/45686269/53d261ed-aad7-4754-913e-891158e82a88)

Reply:
Also, according to the design, there is an overlay on the entire body to slightly darken the background.

![Screenshot 2024-07-02 at 10 17 23](https://github.com/thedxberience/dxberience-frontend-v2/assets/45686269/22c65663-9872-45f7-8048-54b60bb7d1dd)